package.json

```diff
@@ -87,6 +87,7 @@
     "storybook-django": "^0.3.0",
     "stylelint": "^14.2.0",
     "tailwindcss": "^3.0.23",
+    "tailwindcss-vanilla-rtl": "^0.1.0",
```

Comment:
no issues with this I can see but just trying to understand why we did not opt for a more popular package instead of building (aka maintaining) our own through core team.

I can kind of see in the docs, it uses the `l/r` (left/right) chars instead of `s/e` (start/end).

Just trying to understand, as each package we have to maintain is a potential liability, but also each package we use is also that haha.
https://www.npmjs.com/package/tailwindcss-rtl

Reply:
Ok. That makes sense - thanks for explaining.